scripts/collect-i18n-general.ts

```diff
@@ -1,12 +1,12 @@
 import * as fs from 'fs'
 
+import { DESKTOP_DIALOGS } from '../apps/desktop-ui/src/constants/desktopDialogs'
```

Comment:
Want to add a path in the root tsconfig?
Maybe `'@desktop/*': './apps/desktop-ui/src'`